src/main/node-shell-session.ts

```diff
@@ -50,10 +50,19 @@ export class NodeShellSession extends ShellSession {
   }
 
   protected exit(code = 1000) {
+    let timeout;
+
     if (this.podId) {
       this.deleteNodeShellPod();
     }
-    super.exit(code);
+
+    if (code != 1000) {
+      this.sendResponse("Error occurred. Terminal will auto-close in 1 seconds ...");
+      timeout = 1*1000;
```

Comment:
I don't think we should auto close the terminal. Saying an error occurred (with the error code). Should be good enough.

Reply:
@Nokel81 Could you please elaborate, do you mean to print 1001 error code to a terminal?

Reply:
@Nokel81 Yeah..I tried to do it, but it seems that all next functions e.g. `closeWebsocketOnProcessExit();` also are run if we have the timeout > 1s. I can see two identical messages on my terminal, I mean `Terminal will auto-close in 1 seconds`.

Reply:
If it is a duplicate wouldn't not having this message here then work or should we just always auto-close in 15s and not print this message?

Reply:
I found one solution.. We don't need to override parent's `exit()` because in this case we call `exit()` before `closeWebsocketOnProcessExit()`. And message `Terminal will auto-close in 15 seconds` won't be shown.  Now it looks like:
<img width="973" alt="Screen Shot 2021-03-01 at 6 21 14 PM" src="https://user-images.githubusercontent.com/38247153/109521163-729df580-7ac6-11eb-80e4-9247df78f558.png">